File: beginner_tutorials/src/talker.cpp
#include "ros/ros.h"
#include "std_msgs/String.h"
#include <sstream>

int main (int argc, char **argv)
{
    // Initiliaze the node by the given name
    ros::init(argc, argv, "talker");
    ros::NodeHandle n;

    // Initiliaze the publisher and topic properties
    ros::Publisher publisher = n.advertise<std_msgs::String>("chatter", 1000);
    // Run the node in given Hz
    ros::Rate loop_rate(10);

    int count = 0;
    while (ros::ok())
    {
        std_msgs::String msg;
        std::stringstream ss;

        ss << "hello world " << count;
        msg.data = ss.str();

        publisher.publish(msg);
        ros::spinOnce();
        loop_rate.sleep();
        ++count;
    }
    return 0;
}

File: beginner_tutorials/cmd/talker.py
import rospy
from std_msgs.msg import String

def talker():
    rospy.init_node('talker')
    publisher = rospy.Publisher('chatter', String, queue_size=10)
    rate = rospy.Rate(10)

    while not rospy.is_shutdown():
        msg = 'hello world %s' % rospy.get_time()
        publisher.publish(msg)
        rate.sleep()

if __name__ == '__main__':
    try:
        talker()
    except rospy.ROSInterruptException:
        pass